modules/ROOT/pages/waf-rulesets.adoc

```diff
@@ -59,7 +59,7 @@ You can understand the regular expressions that make up each rule by viewing the
 
 == Runtime Fabric Policy Summary
 
-The Runtime Fabric policy summary includes the WAF summary. A WAF summary log message is generated every five minutes. See xref:1.0@runtime-fabric::configure-log-forwarding[Configure Log Forwarding on Runtime Fabric]. The contents in the raw log message include WAF and DoS summary statistics.
+The Runtime Fabric policy summary includes the WAF summary. A WAF summary log message is generated every five minutes. See xref:runtime-fabric::configure-log-forwarding[Configure Log Forwarding on Runtime Fabric]. The contents in the raw log message include WAF and DoS summary statistics.
```

Comment:
With the 4/20 release, the summary log messages will be generated every minute. We may change this back to five minutes so it's up to you if you want to leave this as five minutes or change it to one minute.

Reply:
I changed it to one minute